R/class-plotdata-map-markers.R

```diff
@@ -29,6 +27,13 @@ newMapMarkersPD <- function(.dt = data.table::data.table(),
   lat <- veupathUtils::findColNamesFromPlotRef(variables, 'latitude')
   lon <- veupathUtils::findColNamesFromPlotRef(variables, 'longitude')
 
+  # for the others this happen in the parent class, but color is different here
```

Comment:
why is this different? Other than we've assigned the colors to the x axis?

Reply:
just that i think, but  i didnt see that there was an obviously correct and cost effective way to sort that out

Reply:
cool. I have no helpful suggestions, i just wanted to be sure i understood :)